### PR TITLE
update faucet docs

### LINKED ISSUE #24: Add documentation for Local and Sepolia Faucets
## Issue Overview

We have added a faucet feature to address gas issues that developers might encounter. Currently we do not have any documentation on this feature yet.

## Proposed Solutions or Ideas

- Your documentation should be clear and concise.
- Use images where appropriate
- Use proper styling and hierarchy for text
- Use proper styling for code and variable names.
- Use proper callouts when required

Refer to docusaurus for more info: https://docusaurus.io/

### PR DESCRIPTION
**Pull request by pheobeayo**
# Add documentation for Local and Sepolia Faucets

Fixes #24 

## Types of change

- [ ] Enhancement
- [ ] Documentations with pictures references

### CODE REVIEW

**Review by metalboyrick — CHANGES_REQUESTED** [review]
Hi @pheobeayo, really appreciate your PR! 

However I think you've misunderstood the issue, what i meant is explaining to users that we have a faucet feature that they can use in devnet to fill in their wallet in case they need any gas funds, and the sepolia faucet info that will lead them into the external starknet sepolia faucets. 

You can access the faucet on the bottom of the scaffold stark app.

Thank you!

**Comment by pheobeayo** [comment]
okay , will make the changes

File: docs/deploying/deploy-faucets.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+On clicking on it, You will get a pop up
+
+![Sepolia Example](/img/Sepolia.jpg)
+
+
+## Websites available for Starknet Sepolia faucets
+
+These three options are available for any user to get Starknet Sepolia Faucets
+
+### [Starknet Foundation](https://starknet-faucet.vercel.app/)
```
Comment (metalboyrick) [inline docs/deploying/deploy-faucets.mdx:24]: This can be bullet points, no headers here.

File: docs/deploying/deploy-faucets.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 3
+toc_max_heading_level: 5
+---
+
+# Faucets (Available)
```
Comment (metalboyrick) [inline docs/deploying/deploy-faucets.mdx:6]: We can just say "Faucets"

File: static/img/calldata.jpg
Comment (metalboyrick) [inline static/img/calldata.jpg:1]: We can delete this data if not used.

File: docs/deploying/deploy-faucets.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Faucets for deployment of smart contracts can be requested from the website.
+
+The button is at the bottom left of the  [Demo app] (https://scaffold-stark-demo.vercel.app/debug)
```
Comment (metalboyrick) [inline docs/deploying/deploy-faucets.mdx:10]: This hyperlink doesn't show what you've intended I think. Can check the syntax again.

image

File: docs/deploying/deploy-faucets.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+
+![Faucets Example](/img/faucets.jpg)
+
+On clicking on it, You will get a pop up
+
+![Sepolia Example](/img/Sepolia.jpg)
+
+
+## Websites available for Starknet Sepolia faucets
```
Comment (metalboyrick) [inline docs/deploying/deploy-faucets.mdx:20]: We also need some docs on the Devnet faucet, which looks different from this sepolia window. The devnet one allows user to fill in their own wallet with some tokens

File: yarn.lock
Comment (metalboyrick) [inline yarn.lock:1]: Just curious how did this get updated without updating `package.json`

**Review by metalboyrick — CHANGES_REQUESTED** [review]
Left some comments! Don't worry you still have some time to make the changes! All the best!

File: docs/deploying/deploy-faucets.mdx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+These three options are available for any user to get Starknet Sepolia Faucets
+
+### [Starknet Foundation](https://starknet-faucet.vercel.app/)
+
+### [Alchemy](https://www.alchemy.com/faucets/starknet-sepolia)
+
+### [Blast](https://blastapi.io/faucets/starknet-sepolia-eth)
+
+#### In  accessing any of these websites, you will paste or input your wallet address
```
Comment (metalboyrick) [inline docs/deploying/deploy-faucets.mdx:30]: I think we can remove this part.

File: docs/deploying/deploy-faucets.mdx (hunk 4 above)
Comment (metalboyrick) [reply docs/deploying/deploy-faucets.mdx:20]: For this section, we can just say "Starknet Sepolia Faucets"

**Review by metalboyrick — CHANGES_REQUESTED** [review]

**Comment by Nadai2010** [comment]
Hi @pheobeayo great jobs, thank you, some details

- Check error in [Demo app] (https://scaffold-stark-demo.vercel.app/) vs [Demo app](https://scaffold-stark-demo.vercel.app/)
- Add the "*" that @metalboyrick mentioned, similar and consistent
imagen

- You must indicate in devnet that you can transfer faucet.
- 
- Better define a account address and amount as an example and 1 ETH,

```bash
curl -d '{"amount":1000000000000000000, "address":"0x064b48806902a367c8598f4f95c305e8c1a1acba5f082d294a43793113115691"}' -H "Content-Type: application/json" -X POST http://localhost:5050/mint
```

File: docs/deploying/deploy-faucets.mdx (hunk 3 above)
Comment (metalboyrick) [reply docs/deploying/deploy-faucets.mdx]: @pheobeayo This is not resolved yet, you can try editing while running the localhost instance with `yarn start`. The vscode markdown doesn't necessarily give you the right representation.

File: docs/deploying/deploy-faucets.mdx (hunk 4 above)
Comment (metalboyrick) [reply docs/deploying/deploy-faucets.mdx:20]: @pheobeayo I meant the title could just be "Starknet Sepolia Faucets" instead of "Websites available ...."

Comment (metalboyrick) [reply docs/deploying/deploy-faucets.mdx:20]: and can remove the sub "Starknet Sepolia Faucets"

File: docs/deploying/deploy-faucets.mdx (hunk 1 above)
Comment (metalboyrick) [reply docs/deploying/deploy-faucets.mdx:24]: @pheobeayo not resolved yet

File: docs/deploying/deploy-faucets.mdx (hunk 4 above)
Comment (metalboyrick) [reply docs/deploying/deploy-faucets.mdx:20]: Also for the devnet, please indicate the UI path also. as shown by @Nadai2010

**Comment by metalboyrick** [comment]
Hey @pheobeayo, some of the changes are still not addressed, feel free to ping us should you have any questions!

**Comment by pheobeayo** [comment]
@Nadai2010, the picture was added for the Devnet local faucets. Where can I get that, please?

@metalboyrick The link in the Demo app which isn't not giving the right link is actually giving the right link with me wherever I click on it, Please what could be wrong?

**Comment by Nadai2010** [comment]
Hi @pheobeayo,  I think you can use image hat I shared , if you want to access the faucet on devnet, you should follow the [Scaffold Stark](https://github.com/Quantum3-Labs/scaffold-stark-2) repository and follow the steps to lift the UI

**Comment by metalboyrick** [comment]
@pheobeayo the devnet can only be accessed if you make a local copy of the repo. you cannot use the demo link for that

**Comment by pheobeayo** [comment]
Yes, I understand, I did my research too, Kindly review the changes I made on the PR @metalboyrick

**Comment by pheobeayo** [comment]
Thank you for the support all along, it was great contributing to this! @metalboyrick  @Nadai2010

**Pull request merged**